shub/config.py

```diff
@@ -28,6 +28,8 @@ def __init__(self):
         }
         self.apikeys = {}
         self.version = 'AUTO'
+        self.stacks = {}
+        self.requirements_file = None
```

Comment:
This was probably discussed elsewhere before: Why not default to `requirements.txt`? I'd think that's what most users expect, and it would facilitate a care-free `shub deploy`.